kube/stateful_set.go

```diff
@@ -36,7 +36,7 @@ func NewStatefulSet(role *model.Role, settings *ExportSettings) (helm.Node, helm
 	statefulSet.Add("spec", spec)
 	err = replicaCheck(role, statefulSet, svcList, settings)
 
-	return statefulSet.Sort(), svcList, err
+	return statefulSet, svcList, err
```

Comment:
Why isn't this being sorted any more? It's not clear to me why this is being done, I thought the problem was with deployments only.

Reply:
Oh, nevermind, I see now that it was moved to the common `replicaCheck` function.